authors/apps/articles/views.py

```diff
@@ -550,7 +550,7 @@ def post(self, request, slug):
             serializer.is_valid(raise_exception=True)
             serializer.save()
             return Response({"message":
-                             "Article rated."},
+                             "You have rated this article: " + slug},
```

Comment:
@RachelleMaina, you do not need to add a slug to the article return message. You can return JSON object with a field of article having key of `article` and value of it's slug together with the success message. example shown below:

```
{
    "article": "how-to-train-your-dragon"
   "message":  "Article rated succesfully",
}
```